assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -288,6 +288,7 @@ storage:
           %{~ if controller_count == "1" ~}
           -A INPUT -p tcp --dport 7443 -j ACCEPT
           %{~ endif }
+          -A INPUT -p tcp --dport 9100 -j ACCEPT
```

Comment:
This opens the port on all interfaces, but node-exported only listens on host IP? So it's not accessible from public, right?

Reply:
It seems https://github.com/kinvolk/lokomotive/issues/137 becomes more pressing :/

Reply:
With this change, the controller is exposed publicly:

```console
$ curl $controller:9100
<html>
                        <head><title>Node Exporter</title></head>
                        <body>
                        <h1>Node Exporter</h1>
                        <p><a href="/metrics">Metrics</a></p>
                        </body>
                        </html>
```

Reply:
Workers timeout though.

Reply:
I am surprised that this overrides the calico policies. Wondering how this can be fixed in the right way.

Reply:
Blocked merging with change request, to avoid accidental merging.

Reply:
@invidian if this works https://github.com/helm/charts/pull/21490 then we can incorporate upstream chart here. Otherwise we will have to add this change locally and keep maintaining it.

Reply:
Yay the PR is merged!